src/components/search/SimpleSearchRow.tsx

```diff
@@ -0,0 +1,97 @@
+import { Grid } from '@mui/material'
+import { useState } from 'react'
+import { DisplayField } from '../../types/Fields'
+import { SearchParameter } from '../../types/SimpleSearch'
+import SearchDateInput from './SearchDateInput'
+import SearchDropdown from './SearchSelectField'
+import SearchTextInput from './SearchTextInput'
+
+interface SimpleSearchRowProps {
+  field: DisplayField
+  parameter: SearchParameter
+  index: number
+  onChange: (e: React.ChangeEvent<any>) => void
+}
+
+const SimpleSearchRow: React.FC<SimpleSearchRowProps> = ({
+  field,
+  parameter,
+  index,
+  onChange
+}) => {
+  
+  const options = [
+    'Exact',
+    'Low Fuzziness',
+    'Medium Fuzziness',
+    'High Fuzziness'
+  ]
```

Comment:
Let's have the options rather an array of objects :
```suggestion
  const options = [
    { value: 0, label: 'Exact' },
    { value: 1, label: 'Low Fuzziness'},
    { value: 2, label: 'Medium Fuzziness'},
    { value: 3, label: 'High Fuzziness'}
  }]
```

Reply:
yay. Thanks